torch/_dynamo/output_graph.py

```diff
@@ -1577,14 +1577,19 @@ def update_used_symbols(used_symbols, fake: Union[torch.SymInt, torch.Tensor]):
                     if isinstance(node.meta["grapharg"].example, torch.ScriptObject):
                         real_script_obj = node.meta["grapharg"].example
                         fake_script_obj = node.meta["grapharg"].example_strong_ref
-                        flat_dict = dict(real_script_obj.__obj_flatten__())  # type: ignore[attr-defined]
-                        for attr in flat_dict.keys():
-                            fake_attr_val = getattr(fake_script_obj.wrapped_obj, attr)
-                            pytree.tree_map_only(
-                                (torch.SymInt, torch.Tensor),
-                                lambda t: update_used_symbols(used_symbols, t),
-                                fake_attr_val,
-                            )
+                        if not torch._library.fake_class_registry.tracing_with_real(
```

Comment:
Tracing with real will always gives static shape and there is no way we could peek into its contents and get all the symbols. So we can just skip.